MerlinAU.sh

```diff
@@ -2788,8 +2788,10 @@ Please manually update to version $minimum_supported_version or higher to use th
     check_memory_and_prompt_reboot "$required_space_kb" "$availableRAM_kb"
 
     # Extracting the firmware binary image #
-    if unzip -o "$FW_ZIP_FPATH" -d "$FW_BIN_DIR" -x README*
-    then
+    if output=$(unzip -o "$FW_ZIP_FPATH" -d "$FW_BIN_DIR" -x README* 2>&1); then
+        echo "$output" | while IFS= read -r line; do
+            Say "$line"
+        done
```

Comment:
Very good idea. If something goes wrong at this step, it may help to have the output in the logs.